### ISSUE TITLE
Failed to upgrade from 19.6 to 19.7 on CentOS 6.7 (el6) due to low glibc version 2.12

### ISSUE THREAD

**Issue by huang--yijian** [ClickHouse/ClickHouse#7998]
**Describe the bug or unexpected behaviour**
When we follow official instructions to upgrade ClickHouse from 19.6 to 19.17 on a CentOS release 6.7 (el/6) x86_64 machine, it fails by the following error:

Package: clickhouse-common-static-19.17.4.11-2.x86_64 (repo.yandex.ru_clickhouse_rpm_stable_x86_64)
          Requires: libc.so.6(GLIBC_2.16)(64bit)

![image](https://user-images.githubusercontent.com/7484190/70035124-4490e380-15ed-11ea-85d0-4520f7ddfa5c.png)

**How to reproduce**
Install ClickHouse 19.17.4.11 by official instructions using yum install, on a CentOS 6.7 (el/6) machine with glibc 2.12, the built-in glibc version of CentOS 6.7.

* Which ClickHouse server version to use
We try to upgrade from 19.6 to 19.17.4.11

**Expected behavior**
The upgrade should succeed without any error.

**Comment by huang--yijian** [comment]
Also Altinity does not provide RPM of ClickHouse 19.17 for el/6.

**Comment by nicelulu** [comment]
try ```ldconfig -p |grep libc.so``` cc.

**Comment by alexey-milovidov** [comment]
The binary itself does not require this libc version:
```
$ ldd -v /usr/bin/clickhouse
        linux-vdso.so.1 (0x00007ffda6934000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fbdbd2c6000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007fbdbd2bc000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fbdbd2b6000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fbdbd0cc000)
        /lib64/ld-linux-x86-64.so.2 (0x00005613d9217000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fbdbcf3f000)

        Version information:
        /usr/bin/clickhouse:
                libc.so.6 (GLIBC_2.4) => /lib/x86_64-linux-gnu/libc.so.6
                libc.so.6 (GLIBC_2.2.5) => /lib/x86_64-linux-gnu/libc.so.6
                libc.so.6 (GLIBC_2.3) => /lib/x86_64-linux-gnu/libc.so.6
                libc.so.6 (GLIBC_2.3.2) => /lib/x86_64-linux-gnu/libc.so.6
                libc.so.6 (GLIBC_2.3.4) => /lib/x86_64-linux-gnu/libc.so.6
                librt.so.1 (GLIBC_2.2.5) => /lib/x86_64-linux-gnu/librt.so.1
                librt.so.1 (GLIBC_2.3.3) => /lib/x86_64-linux-gnu/librt.so.1
                ld-linux-x86-64.so.2 (GLIBC_2.3) => /lib64/ld-linux-x86-64.so.2
                libpthread.so.0 (GLIBC_2.2.5) => /lib/x86_64-linux-gnu/libpthread.so.0
                libpthread.so.0 (GLIBC_2.3.4) => /lib/x86_64-linux-gnu/libpthread.so.0
                libpthread.so.0 (GLIBC_2.3.2) => /lib/x86_64-linux-gnu/libpthread.so.0
                libdl.so.2 (GLIBC_2.2.5) => /lib/x86_64-linux-gnu/libdl.so.2
                libm.so.6 (GLIBC_2.2.5) => /lib/x86_64-linux-gnu/libm.so.6
```

And the package also should not.

@alesapin will help to figure out why the package has wrong dependencies information.

**Comment by huang--yijian** [comment]
> ldconfig -p |grep libc.so

`[root@storage10 ~]# ldconfig -p |grep libc.so
	libc.so.6 (libc6,x86-64, OS ABI: Linux 2.6.18) => /lib64/libc.so.6`

![image](https://user-images.githubusercontent.com/7484190/70110349-a30b9f80-1689-11ea-86cc-f9a6e1f67a22.png)

**Comment by alesapin** [comment]
Dependency comes from `clickhouse-odbc-bridge` :(
```
$ readelf -s clickhouse-odbc-bridge | grep  '@GLIBC_' | grep '2.16'
   120: 0000000000000000     0 FUNC    GLOBAL DEFAULT  UND aligned_alloc@GLIBC_2.16 (6)
```

**Comment by alesapin** [comment]
Fix https://github.com/ClickHouse/ClickHouse/pull/8028.

**Comment by alesapin** [comment]
Merged, we will backport this fix to 19.17, 19.16 and 19.15 if possible.

**Comment by huang--yijian** [comment]
> Merged, we will backport this fix to 19.17, 19.16 and 19.15 if possible.

Thank you for the quick support.
When the fix is done, please notify us.

**Comment by huang--yijian** [comment]
> Merged, we will backport this fix to 19.17, 19.16 and 19.15 if possible.

Thank you for the quick support.
When the fix is released in the yum packages to install, please notify us.
